lib/livebook/file_system.ex

```diff
@@ -33,6 +33,8 @@ defprotocol Livebook.FileSystem do
   """
   @type error :: String.t()
 
+  @type t :: Livebook.FileSystem.S3.t()
+
```

Comment:
Should also be local, but this is a protocol, so it already defines `t` :)

```suggestion
```

Reply:
TIL